services/frontend/src/components/admin/dashboardGraphs/DashboardGraphsView.jsx

```diff
@@ -138,7 +123,9 @@ const DashboardGraphsView = ({
               }
               loading={monthlyGrowth.length === 0}
             >
-              <Chart
+              <Chart type="line" data={monthlyGrowthData} options={options} />
+
+              {/* <Chart
```

Comment:
Looks good, I would remove this comment and design wise for the charts, we can work on it after this is merged, unless you want to do it now

Reply:
I think we can work on the design after this merge just to test out chart js first